### PR TITLE
More corrections to AWS procedures

### PR DESCRIPTION
**Pull request by ianf77**
Changes to existing procedures

### CODE REVIEW

**Review by itdove — APPROVED** [review]

File: stories/topics/proc-aws-marketplace.adoc [hunk 1]
```diff
@@ -8,18 +8,46 @@ Follow these steps to purchase and deploy the public offer.
...
+.. In the *Specify template* section under *Template source* click btn:[Upload a template file] and select the template you have modified.
+..Click btn:[Next] to go to the next step.
+.. Enter a stack name and select an EC2 KeyPair to use for the deployment.
+.. For *Select VPC*, select *New* or *Existing*. For more information, see xref:con-gcp-application-deploy[Application deployment].
+.. Click btn:[Next] to go to the *Configure stack options* page.
+.. Optional: Configure stack options.
+.. Click btn:[Next] to review the stack.
+.. Click btn:[Submit] to create your new deployment with the updated CIDR range for the load balancers.
+
+<<<<<<< HEAD
```
Comment (gebhardtr) [inline stories/topics/proc-aws-marketplace.adoc:45]: some leftover merge artifacts

File: stories/topics/proc-aws-deploy-existing-vpc.adoc [hunk 2]
```diff
@@ -23,6 +23,9 @@ This launches the AWS CloudFormation Console in a new browser tab.
 ** *Existing VPC CIDR Range* 
 ** *Existing Private Subnet 1 ID*
 ** *Existing Private Subnet 2 ID*
++
+Note that copying and pasting these from the AWS console introduces hidden characters, and cause the entries to fail.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-deploy-existing-vpc.adoc:27]:
```suggestion
Note that copying and pasting these from the AWS console introduces hidden characters, and causes the entries to fail.
```

File: stories/topics/proc-aws-marketplace.adoc [hunk 3]
```diff
@@ -8,17 +8,39 @@ Follow these steps to purchase and deploy the public offer.
...
 . Click btn:[Continue to Configuration].
 Select the appropriate fulfillment options.
 Note that some selectors may have only one option.
+. In the *Fulfillment option* field, ensure *{PlatformNameShort} CloudFormations Topology* is selected.
+. In the *Software version* field, select the latest available version from the list.
+. In the *Region* field, select the region with your EC2 key pair. 
+The CloudFormation stack deploys in the same region.
 . Click btn:[Continue to Launch].
-
+. In the *Choose Action* filed, select *Launch CloudFormation*.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-marketplace.adoc:24]:
```suggestion
. In the *Choose Action* field, select *Launch CloudFormation*.
```

File: stories/topics/proc-aws-marketplace.adoc [hunk 4]
```diff
@@ -8,17 +8,39 @@ Follow these steps to purchase and deploy the public offer.
...
+. Click btn:[Save].
+. Select *Local File* and supply a name.
+. In your saved template locate the `controlleralbsecuritygroupEFD044E7` and `hubalbsecuritygroup4C1C53F8` CIDR ranges. 
+In *SecurityGroupIngress* change the CidlrIP to your desired CIDR range.
+. Create a new {AAPonAWS} CloudFormation stack using the modified CIDR range template for the load balancers.
+.. In the AWS Console, navigate to menu:CloudFormation[].
+.. Click btn:[Create Stack].
+.. Select *With new resources (standard)* from the dropdown.
+.. In the *Specify template* section under *Template source* click btn:[Upload a template file] and select the template you have modified.
+..Click btn:[Next] to go to the next step.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-marketplace.adoc:37]:
```suggestion
.. Click btn:[Next] to go to the next step.
```

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**